p2p/raintree/network.go

```diff
@@ -122,24 +139,23 @@ func (n *rainTreeNetwork) NetworkSend(data []byte, address cryptoPocket.Address)
 	return n.networkSendInternal(bz, address)
 }
 
+// networkSendInternal sends `data` to the peer at pokt `address` if not self.
 func (n *rainTreeNetwork) networkSendInternal(data []byte, address cryptoPocket.Address) error {
 	// NOOP: Trying to send a message to self
+	n.logger.Debug().Str("self", n.selfAddr.String()).Str("target", address.String()).Msg("HEER!!!!")
```

Comment:
```suggestion
	n.logger.Debug().Str("self", n.selfAddr.String()).Str("target", address.String()).Msg("About to send message via RainTree")
```

Reply:
Hehe, nice catch - thanks! 🙌😅

IIRC I was using this to debug and missy have gotten a little sloppy. My apologies. Unless you feel strongly about keeping this line, I would prefer to drop it as it will be very noisy (for no good reason IMO) in the logs.